content/applications/finance/accounting/fiscal_localizations/localizations/chile.rst

```diff
@@ -499,22 +507,35 @@ Common Errors
 There are multiple reasons behind a rejection from the SII, but these are some of the common errors
 you might have and which is the related solution. 
 
-- Error: RECHAZO- DTE Sin Comuna Origen.
+- Error: ``RECHAZO- DTE Sin Comuna Origen.``
   
   *Hint:* Make sure the Company Address is properly filled including the State and City. 
 
-- Error en Monto :  - IVA debe declararse.
+- Error en Monto: ``- IVA debe declararse.``
   
   *Hint:* The invoice lines should include one VAT tax, make sure you add one on each invoice line. 
 
-- Error: Rut No Autorizado a Firmar.
+- Error: ``Rut No Autorizado a Firmar.``
   
   *Hint:* The invoice lines should include one VAT tax, make sure you add one on each invoice line. 
 
-- Error: Fecha/Número Resolucion Invalido RECHAZO- CAF Vencido : (Firma_DTE[AAAA-MM-DD] - 
-  CAF[AAAA-MM-DD]) &gt; 6 meses.
+- Error: ``Fecha/Número Resolucion Invalido RECHAZO- CAF Vencido : (Firma_DTE[AAAA-MM-DD] -
+  CAF[AAAA-MM-DD]) &gt; 6 meses.``
   
   *Hint:* Try to add a new CAF related to this document as the one you’re using is expired.
+  
+- Error: ``Element '{http://www.sii.cl/SiiDte%7DRutReceptor': This element is not expected. Expected
+  is ( {http://www.sii.cl/SiiDte%7DRutEnvia ).``
```

Comment:
The formatting of this term seems odd, is it normal that there's an opening brace but no closing brace? Is that what the error message actually contains?

Reply:
@ren-odoo ?

Reply:
Hi @xmo-odoo @jcs-odoo In this case I just added the message exactly as we receive it from the Chilean Government website. I know it's odd but if we change it, it won't longer match with the message users will see in the invoice chatter.